src/modules/languages/typst.nix

```diff
@@ -13,6 +13,14 @@ in
       default = pkgs.typst;
       defaultText = lib.literalExpression "pkgs.typst";
     };
+
+    fontPaths = lib.mkOption {
+      type = lib.types.listOf lib.types.str;
+      description = "Directories to be searched for fonts.";
+      default = [ ];
+      defaultText = lib.literalExpression "[]";
+      example = [ "\${pkgs.roboto}/share/fonts/truetype" ];
```

Comment:
```suggestion
      example = lib.literalExpression [ "\${pkgs.roboto}/share/fonts/truetype" ];
```

Reply:
I don't understand what lib.literalExpression does so I'll trust you on this.